index.html

```diff
@@ -4,60 +4,76 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mohamed Aden Ighe: My personal portfolio</title>
     <link rel="stylesheet" href="style.css">
     <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet"/>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/> 
+</head>
 
-    <title>Mohamed Aden Ighe: My personal portfolio</title>
+<body>    
+<header class="header">
 
-<body>
+  <!-- Header -->
+  <div class="container">
     <header id="part1">
-        <div class ="logo-placeholder">
-            <a href="#about"><p class="logo">MAI</p></a>
-        </div>
+        <nav class="logo-placeholder">
+            <a href="" class="logo"">MAI</a>
```

Comment:
[OPTIONAL]
- [ ] For the best semantic structure, kindly note that the `header` cannot be a descendant of a child to another element, also, you have already used the `header` as the main wrapper on line `14`, kindly replace it with a `div`.